internal/shellgen/tmpl/flake_remove_nixpkgs.nix.tmpl

```diff
@@ -17,6 +17,14 @@
    }:
       let
         pkgs = nixpkgs.legacyPackages.{{ .System }};
+        {{- range .FlakeInputs }}
+        {{- if .IsNixpkgs }}
+        {{.PkgImportName}} = (import {{.Name}} {
+          system = "{{ $.System }}";
+          config.allowUnfree = true;
+        });
+        {{- end }}
+        {{- end }}
```

Comment:
these lines are the same as in the current `flake.nix`:
https://github.com/jetpack-io/devbox/blob/main/internal/shellgen/tmpl/flake.nix.tmpl#L27-L34